frontend/src/static/js/main.js

```diff
@@ -250,7 +250,11 @@ window.app = new window.Vue({
       return JSON.stringify(dataObj);
     },
 
-    getRepoSenseLink() {
+    getRepoSenseHomeLink() {
+      return 'https://github.com/reposense/RepoSense';
```

Comment:
Should be `reposense.org`?

Reply:
Have updated the link, but wished to understand the difference since both the links open the same page for me. Is it that the former link could have changed with time but the present one will continue to remain the same?

Reply:
`reposense.org` is the product website while the github link is the dev website. At the moment former points to the latter, but eventually it will be a separate website, such as https://markbind.org/